microsoft-edge/webview2/samples/webview2-winui3-sample.md

```diff
@@ -0,0 +1,603 @@
+
+To commit and push changes that were made to the sample in your working branch:
+
+1. In Visual Studio, select **File** > **Save All** (Ctrl+Shift+S).
+
+1. In GitHub Desktop, enter a brief commit message, such as "updated, built, ran sample", and then click the **Commit** button.
+
+1. Select **Repository** > **Push**.  Your working branch is copied from your local drive to your forked branch of the WebView2Samples repo at GitHub.
+
+   **View your working branch at GitHub**
+
+1. In Microsoft Edge, go to your fork of the repo, such as `https://github.com/myGithubUsername/WebView2Samples/`.
+
+1. Click the **[#] Branches** link, and then select your new working branch, such as `user/myGithubUsername/winui3-sample`.
+
+
+<!-- ====================================================================== -->
+## Step 8: Inspect the code
+
+1. Close the **WebView2_WinUI3_Sample (Package)** app.
+
+1. In Solution Explorer, in the **WebView2_WinUI3_Sample** project, double-click each `.xaml` file and its `.cs` file:
+   * `App.xaml` - No WebView2 code.
+      * `App.cs` - Code to use a fixed-version WebView2 Runtime.
+   * `MainWindow.xaml` - WebView2 elements.
+      * `MainWindow.cs` - WebView2 code.
+
+Condensed versions of these files are below.
+
+
+<!-- ------------------------------ -->
+#### App.xaml
+
+There's no WebView2-specific code in this file.
+
+
+<!-- ---------- -->
+###### Condensed listing
+
+```xaml
+<Application
+    x:Class="WebView2_WinUI3_Sample.App"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:WebView2_WinUI3_Sample">
+    <Application.Resources>
+        <ResourceDictionary>
+            <ResourceDictionary.MergedDictionaries>
+                <XamlControlsResources xmlns="using:Microsoft.UI.Xaml.Controls" />
+            </ResourceDictionary.MergedDictionaries>
+        </ResourceDictionary>
+    </Application.Resources>
+</Application>
+```
+
+
+<!-- ------------------------------ -->
+#### App.cs
+
+<!-- todo: create PR in Samples repo to improve comment block like below -->
+
+The only WebView2-specific code in this file is commented out.  You can un-comment these lines in order to use a fixed-version Runtime; see [Using a fixed-version WebView2 Runtime](#using-a-fixed-version-webview2-runtime), below.
+
+
+<!-- ---------- -->
+###### Condensed listing
+
+```csharp
+namespace WebView2_WinUI3_Sample
+{
+    /// <summary>
+    /// Provides application-specific behavior to supplement the default Application class.
+    /// </summary>
+    public partial class App : Application
+    {
+        /// <summary>
+        /// Initializes the singleton application object.  Equivalent to main() or WinMain().
+        /// </summary>
+        public App()
+        {
+            this.InitializeComponent();
+            // If you're shipping a fixed-version WebView2 Runtime with your app,
```

Comment:
i plan to create PR in WebView2Samples repo to update the spelling/ wording/ formatting of this entire comment block
@peiche-jessica

Reply:
use better version # than 95.x?

Reply:
Here's a link to [PR 258](https://github.com/MicrosoftEdge/WebView2Samples/pull/258) (WinUI 3 sample: Update Readme and fixed-version Runtime code) in WebView2Samples repo.